<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Display</class>
 <widget class="QMainWindow" name="Display">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>526</width>
    <height>352</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Display</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>581</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_logo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>40</y>
        <width>211</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(./img/smartphone.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>30</y>
        <width>141</width>
        <height>263</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Serveur addresse</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_addr"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Serveur port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_port_server"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Client port</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_port_client"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Pseudonyme</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="input_name"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_connexion">
         <property name="text">
          <string>Connexion</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>270</width>
        <height>306</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QPushButton" name="pushButton_deconnexion">
           <property name="text">
            <string>Deconnexion</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="listWidget"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="buttunCall">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>61</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>61</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">border-image: url(./img/phone-endcall.png)</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QLabel" name="label_conversation">
	<property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>171</width>
        <height>171</height>
       </rect>
      </property>
       <property name="text">
        <string></string>
       </property>
      </widget>
     <widget class="QPushButton" name="pushButtonQuitCall">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>260</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>71</width>
        <height>71</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: url(./img/phone-call.png)</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
